packages/core/integration-tests/test/html.js

```diff
@@ -2213,7 +2213,7 @@ describe('html', function () {
       },
       {
         type: 'css',
-        assets: ['shared.css'],
+        assets: ['shared.css', 'other.css'], //We've duplicated 'other' to maintain 1 css bundle per entry
```

Comment:
Shouldn't this only apply to bundles with `needsStableName`? In this case, `other.css` is moved into a shared bundle which is fine because it's in HTML and not an entry.

Reply:
Thank you for reviewing! My confusion is when exactly css bundles should be merged. In this example, there are technically only two bundlegroups right? Since the type change bundles are not bundleGroups. So we have a.html and b.html bundleGroup. And I guess my assumption was that all bundles in an entry bundleGroup include `needsStableName` . 

I think that's where I'm unsure of what the behavior should be. cc: @devongovett